src/media/css/detail/content-ratings.styl

```diff
@@ -15,7 +15,13 @@
         width: 48px;
     }
     .infobox-wrapper {
+        display: flex;
+        flex-wrap: wrap;
```

Comment:
It doesn't break functionality though it causes a misalignment in the IARC block so I am willing to leave it be as we're dropping support for 1.1

![](http://i.imgur.com/VdtPfMD.png)